src/bepasty/apis/rest.py

```diff
@@ -239,6 +241,42 @@ def get(self, name):
         return super(ItemDetailView, self).get(name)
 
 
+class ItemDeleteView(DeleteView, RestBase):
+    def error(self, item, error):
+        raise Conflict(description=error)
+
+    def response(self, name):
+        return make_response('', {'Content-Type': 'application/json'})
```

Comment:
how does this response look like exactly? is it valid json?

Reply:
if this needs fixing, fix globally.

Reply:
sure, response.json raise decode error. fixed by using '{}' instead of ''